README.md

```diff
@@ -47,6 +47,109 @@ The main design is to separate execution logic from execution. The execution can
+    @Timeout(100)
+    public Response allSchedules() {
+        final List<Schedule> allSchedules = scheduleDAO.getAllSchedules();
+        final GenericEntity<List<Schedule>> entity = buildEntity(allSchedules);
+        try {
+            Thread.sleep(102);
+        } catch (InterruptedException e) {
+            return Response.status(Response.Status.NOT_FOUND).build();
+        }
+        return Response.ok(entity).build();
+    }
+```
+
+### @CircuitBreaker
+The `@CircuitBreaker` annotation allows you to prevent repeating timeout so that the failing services fail fast. The below code snippet means that the circuit is open once 2 (4 * 0.5) failures occur among the rolling window of 4 consecutive invocations. The circuit will stay open for 1000ms. After 5 consecutive successful invocations of the method, the circuit will be closed. When a circuit is open a `CircuitBreakerOpenException` will be thrown.
+
+```
+	 @PUT
+    @Path("/search")
+    @Counted(monotonic = true,tags="app=speaker")
+    @CircuitBreaker(requestVolumeThreshold=4, failureRatio=0.50, delay=1000, successThreshold=5)
+    public Set<Speaker> searchFailure(final Speaker speaker) {
+        if (isServiceBroken.get()) {
+            throw new RuntimeException("Breaking Service failed!");
+        }
+        final Set<Speaker> speakers = this.speakerDAO.find(speaker);
+        return speakers;
+    }
+
+```
+
+### @Bulkhead
+If you use @Bulkhead, metrics are added for how many concurrent calls to your method are currently executing, how often calls are rejected, how long calls take to return, and how long they spend queued (if you’re also using @Asynchronous). You can use @Bulkhead in conjunction with @Asynchronous. @Asynchronous causes an invocation to be executed by a different thread. The below example shows the usage of @bulkhead annotation, indicating that only 3 concurrent request to the API is allowed.
+
+```
+    @GET
+    @Timed
+    @Metric
+    @Path("/getAllSpeakers")
+    @Counted(name="io.microprofile.showcase.speaker.rest.monotonic.getAllSpeakers.absolute",monotonic = true,tags="app=speaker")
+    @Bulkhead(3)
+    public Collection<Speaker> getAllSpeakers() {
+        final Collection<Speaker> speakers = this.speakerDAO.getAllSpeakers();
+        speakers.forEach(this::addHyperMedia);
+        return speakers;
+    }
+```
+
+### @Fallback
+Fallback annotation allows you to deal with exceptions. The previous annotations increases the success rate of invocation but you cannot eliminate exception. When an exception occurs its wise to fall back to a different operation.A method can be annotated with @Fallback, which means the method will have Fallback policy applied. The fallback method needs to have same signature as the original method signature. @Fallback can be used in conjunction with any other annotation. The below code snippet means when the method failed and retry reaches its maximum retry, the method `fallBackMethodForFailingService` will be invoked. 
+
```

Comment:
- "... eliminate exceptions."
- space between sentences
- "The below code snippet means that when the method reaches it maximum number of retries, the method ..."